SmokeTester.Tests/SmokeTester.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Did you consider .net6?

Reply:
We did, but we wanted to make it updated but still more available than 6. Do you think we should reconsider?

Reply:
Discussed, left 5 for broader build agent support.